tests/i18n_spec.rb

```diff
@@ -1,13 +1,20 @@
 #!/usr/bin/env rspec
+# typed: false
 
 require_relative "test_helper"
 
 require "yast"
 
-include Yast::I18n
 
-module Yast
-  describe I18n do
+
+describe Yast do
+  describe Yast::I18n do
+    include Yast::I18n
+    extend Yast::I18n
+
+    let(:translated) { "translated".freeze }
+    let(:singular) { "untranslated".freeze }
+    let(:plural) { "plural".freeze }
```

Comment:
why constants does not work?

Reply:
Sorbet is stricter than Ruby and thinks that constants defined in one `describe` are not visible in a *sibling* `describe`. I will commit a shorter fix which just moves the constants up one block level.

Reply:
yeap, that will look better